lib/ts/recipe/thirdparty/components/themes/styles.ts

```diff
@@ -42,10 +42,12 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
         providerButton: {
             display: "flex",
             flexDirection: "row",
+            paddingLeft: "0px",
+            paddingRight: "0px",
         },
 
         providerButtonLeft: {
-            width: "34px",
+            width: "40px",
```

Comment:
Is hardcoding to another value the right solution for sure? How have you tested this?

Reply:
So the problem was that all buttons by default have a left padding of 6px, because of this the image for the provider would always seem off center even though it wasnt. Ive removed the left padding and added it to the width to keeo the layout consistent to how it appears now.

I tested it against the playground-auth app since I already had it set up

Reply:
Discuss on call..